kernel/kernel-api/src/main/java/io/delta/kernel/types/StructType.java

```diff
@@ -122,7 +122,7 @@ public Column column(int ordinal) {
     public Column column(String fieldName) {
         Tuple2<StructField, Integer> fieldAndOrdinal = nameToFieldAndOrdinal.get(fieldName);
         System.out.println("Created column " + fieldName + " with ordinal " + fieldAndOrdinal._2);
```

Comment:
Should we have this print? It should at least use the logger?

Reply:
Yeah, these are leftover artifacts from 3.0.0 preview. Few other places of these stdout prints. Removing this.

Reply:
Also removing this method, as it is no longer needed.